stdlib/JuliaSyntaxHighlighting.version

```diff
@@ -0,0 +1,4 @@
+JULIASYNTAXHIGHLIGHTING_BRANCH = main
+JULIASYNTAXHIGHLIGHTING_SHA1 = d0ea83439db25b837b836822d9cdece6b9cc5fbf
+JULIASYNTAXHIGHLIGHTING_GIT_URL := https://github.com/tecosaur/JuliaSyntaxHighlighting.jl.git
```

Comment:
Is the repo private? The link doesn’t work for me https://github.com/tecosaur/JuliaSyntaxHighlighting.jl

Reply:
Ooops, I thought I updated this to https://github.com/julialang/JuliaSyntaxHighlighting.jl :facepalm: